lightning/src/events/mod.rs

```diff
@@ -750,7 +750,12 @@ pub enum Event {
 		/// [`UserConfig::manually_accept_inbound_channels`]: crate::util::config::UserConfig::manually_accept_inbound_channels
 		user_channel_id: u128,
 		/// The reason the channel was closed.
-		reason: ClosureReason
+		reason: ClosureReason,
+		/// Counterparty in the closed channel.
+		counterparty_node_id: Option<PublicKey>,
+		/// Value of the closing channel (sats).
+		channel_capacity: Option<u64>,
```

Comment:
Maybe include the unit as `channel_capacity_sats`?

Reply:
Sure will do that